packages/amplify-provider-awscloudformation/src/configuration-manager.ts

```diff
@@ -91,22 +91,25 @@ export async function configure(context: $TSContext) {
   }
 }
 
-async function enableServerlessContainers(context: $TSContext) {
+export async function enableServerlessContainers(context: $TSContext) {
   const frontend = context.exeInfo.projectConfig.frontend;
   const { config = {} } = context.exeInfo.projectConfig[frontend] || {};
-  // TODO: check for headless mode parameter to avoid the question
```

Comment:
Are you able to rely on `context.exeInfo.inputParams.yes` to determine if you're running headlessly?

Reply:
I did some testing, and `context.exeInfo.inputParams.yes` seems to always be `true`, even if I run just `amplify-dev configure project` without any args passed via command line. There are of course some other args missing that might hint that we're not running headlessly, like `context.exeInfo.inputParams.javascript` being undefined. But is there a better way to tell if it's running headlessly/via script?

Reply:
Hi @Bentheburrito, we only want to use the --yes flag to determine whether this question is asked. We check for `context.exeInfo.inputParams.yes` in the code frequently. Are you certain this is always set to `true`?

Reply:
Hi @jhockett! Yes, `context.exeInfo.inputParams.yes` seems to be set to `true` whenever the user changes [container or profile settings](https://github.com/aws-amplify/amplify-cli/blob/master/packages/amplify-cli/src/config-steps/c0-analyzeProject.ts#L50-L57). Using the `--yes` flag is the only way I could determine whether the user wants to avoid the prompt - though if I've missed a better way to check this, please let me know, and I'd be happy to make the changes! :slightly_smiling_face:

Reply:
@Bentheburrito until the code you linked to gets fixed, you can try using `context?.input?.options?.yes` for the purposes of this PR.